api/app/services/simplified_rag_service.py

```
@@ -669,727 +551,106 @@ async def cleanup(self):
+                "answer": content,
+                "sources": sources_used or [],
                 "response_time": response_time
             }
         except Exception as e:
-            # Handle errors gracefully
+            # start_time is now always defined, so we can calculate error_time directly
             error_time = time.time() - start_time
             logger.error(f"Error processing query: {str(e)}", exc_info=True)
-
             return {
                 "answer": f"I'm sorry, I encountered an error while processing your question. Please try again or rephrase your question.",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)